### PR TITLE
VACMS-7995: Remove "Entity Embed" module from code base

### LINKED ISSUE #7995: Disable Entity Embed module
## Description

This module is no longer in use and can be disabled and removed.

## Acceptance Criteria
- [x] update hook in va_gov_db uninstalls it (see existing pattern)
- [ ] Entity embed is gone

### CMS Team
Please check the team(s) that will do this work. 

- [ ] `CMS Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team ` (leave Sitewide unchecked and check the specific team instead)
  - [x] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product support`
  - [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by netboss**
- Removed the "Entity Embed" module from code base.

## Description

closes #7995 

## Testing done
Passes all tests

## Screenshots
VACMS-7995-Entity-Embed-module-removed

## QA steps

As user administrator
1. Go to `/admin/modules` and:
    - [ ] Enter "entity embed" in the filter to verify and confirm that the module isn't there any more

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team`
  - [x] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product Support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -25167,5 +25089,5 @@
     "platform-overrides": {
         "php": "7.4"
     },
-    "plugin-api-version": "2.2.0"
+    "plugin-api-version": "2.1.0"
```
Comment (swirtSJW) [inline composer.lock:25092]: I don't think we want to be downgrading this.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
This looks good but there is one rollback in the composer.lock that needs fixing.

File: composer.lock (hunk 1 above)
Comment (netboss) [reply composer.lock:25092]: @swirtSJW 
Should I adjust that manually?

**Comment by netboss** [comment]
@swirtSJW 
This one passed all tests and should be ready to merge.

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
Module uninstall has made it to prod, so this is now clear for merge.

**Pull request merged**